icu4c/source/extra/uconv/uconv.vcxproj

```diff
@@ -34,9 +34,11 @@
       <TypeLibraryName>$(OutDir)\uconv.tlb</TypeLibraryName>
     </Midl>
     <ClCompile>
+      <LanguageStandard>stdcpp17</LanguageStandard>
+      <LanguageStandard_C>stdc11</LanguageStandard_C>
       <WarningLevel>Level3</WarningLevel>
       <CompileAs>Default</CompileAs>
-      <DisableLanguageExtensions>true</DisableLanguageExtensions>
+      <DisableLanguageExtensions>false</DisableLanguageExtensions>
```

Comment:
Why does this change? We don't normally want non-standard compiler features.

Reply:
Those changes are required to be able to select a non-default language standard version (see the compile errors from the first CI run of this PR, before changing those flags):

https://developercommunity.visualstudio.com/t/za-stdc11-leads-to-command-line-error-d8016-za-and/1398258